wp-content/themes/pub/wporg-learn-2020/template-parts/component-breadcrumbs.php

```diff
@@ -14,20 +14,35 @@
 	),
 );
 
-// Get information about the post title.
-$cpt_object = get_post_type_object( get_post_type( get_queried_object() ) );
-
-if ( 'lesson-plan' === get_post_type() ) {
+if ( is_post_type_archive() ) {
 	array_push( $crumbs, array(
-		'label' => ucfirst( $cpt_object->labels->name ),
+		'label' => post_type_archive_title( '', false ),
+		'url' => '',
+	));
+} elseif ( is_singular( 'lesson-plan' )
+	|| is_singular( 'wporg_workshop' )
+	|| is_singular( 'course' ) ) {
+	$cpt_object = get_post_type_object( get_post_type() );
```

Comment:
Have tried to look for a WP API to retrieve the post type label of the current post instead of getting a post type object. But I couldn't find it, and `has_archive` still needs to be retrieved from cpt_object later, so cpt_object is retained.